app/views/challenges/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= simple_form_for(@player) do |f| %>
```

Comment:
I feel like we should name this file something uber specific so that we don't incidentally render it whenever trying to build a form - something like completed_post_form or somthing descriptive of the likes